src/main/java/seedu/address/logic/commands/BlacklistListCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.parser.FindCommandParser;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Model;
+
+/**
+ * Lists out all blacklisted clients
+ */
+public class BlacklistListCommand extends BlacklistCommand {
+
+    /**
+     * Instantiates a {@code BlacklistListCommand} object.
+     */
+    public BlacklistListCommand() {
+        super(Index.fromZeroBased(0)); // this is irrelevant but necessary
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+
+        try {
+            // make the find command do the hard work
+            return (new FindCommandParser().parse("find cs/blacklisted")).execute(model);
+        } catch (ParseException pe) {
+            return null; // this will never happen
```

Comment:
By any chance would it be good to have a good error message/log message anyways? In case find command changes or other code is modified? But for now, I agree, it won't happen.